packages/pasteup/package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "@guardian/pasteup",
-  "version": "0.1.0-alpha.1",
+  "version": "1.0.0-alpha.1",
   "description": "Guardian design tokens",
-  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/guardian/dotcom-rendering"
+  },
+  "files": [
```

Comment:
I've seen that when using `npm publish` any files listed in `.gitignore` will be excluded from the packaged tarball. We have `packages/pasteup/src` in our `.gitignore`, with `npm publish` you can add files to this `files` property in the `package.json` to ensure they're still published. Is that what's going on here? If so, do we need to specify the `src` directory in the path? eg. `src/breakpoints.js`